tests/unittests/tests-ipv6_addr/tests-ipv6_addr.c

```diff
@@ -300,10 +304,13 @@ static void test_ipv6_addr_is_loopback_not_loopback(void)
 static void test_ipv6_addr_is_loopback_loopback(void)
 {
     ipv6_addr_t a = IPV6_ADDR_LOOPBACK;
+    const ipv6_addr_t expected = { {
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01
+        }
+    };
 
-    TEST_ASSERT_EQUAL_INT(0, a.u64[0].u64); /* Don't trust the macro ;) */
-    TEST_ASSERT_EQUAL_INT(1, byteorder_ntohll(a.u64[1]));
-    TEST_ASSERT_EQUAL_INT(true, ipv6_addr_is_loopback(&a));
```

Comment:
This test also tests the function `ipv6_addr_is_loopback()` so just using `memcmp()` here somewhat nullifies the purpose of the test.